app/controllers/admin/assignments_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Admin::AssignmentsController < ApplicationController
+    def index
+        @assignment_date = params[:date] ? Date.strptime(params[:date], "%Y-%m-%d") : Date.today
+
+        @assignments = Assignment.where(date: @assignment_date)
+
+        unless params[:date]
+            @past_assignments = Assignment.in_the_past_week
+            @future_assignments = Assignment.in_the_next_week
+        end
+    end
+  
+    def show
```

Comment:
What's this action for?

Reply:
It's supposed to show a page specifically for a single ```Assignment``` but it looks like I forgot to link to it anywhere in the admin.